vipps-report-api-settlement-guide.md

```diff
@@ -77,8 +75,8 @@ Vippsnummer or e-com Merchant Serial Number (MSNs) to a ledger:
 
 ![ledger vs units, one to one](./images/ledger-vs-units-one-to-one.png)
 
-However, for merchants who require it, Vipps have
-limited support for multiple Vippsnummers/e-com MSNs to be settled together.
+However, for merchants who require it, Vipps has
+limited support for multiple Vipps numbers/ecom MSNs to be settled together.
```

Comment:
We do use Vippsnummer in many places, so I don't know why my spellchecker only noticed that one - probably because it was plural. I will use your suggestion.